### PR TITLE
Java8

### PR DESCRIPTION
**Pull request by ladios**
This replaces #125

### CODE REVIEW

**Comment by NathanielMichael** [comment]
Received an error when attempting to install Java 8 on Ubuntu 12.04 LTS with this pull request:

`./oab-java.sh: line 601: dch: command not found`

Installing `devscripts` via apt fixed that error, however I received another afterward:

`
dpkg-buildpackage: source package oracle-java8
dpkg-buildpackage: source version 8.0-1~precise1
dpkg-buildpackage: source changed by root <root@ubuntu>
 dpkg-source --before-build src
dpkg-buildpackage: host architecture amd64
dpkg-checkbuilddeps: Unmet build dependencies: debhelper (>= 9)
dpkg-buildpackage: warning: Build dependencies/conflicts unsatisfied; aborting.
dpkg-buildpackage: warning: (Use -d flag to override.)
51536's retcode: 3
`

I then installed `debhelper` via apt and got another one:

`./oab-java.sh: line 684: rngd: command not found`

Installed `rng-tools` and got everything to work.

**TL;DR** - You need to install `devscripts debhelper rng-tools` via apt to get this to build (for Java8, maybe others as well, I didn't test it).

A lot of dependencies to get this to build... can they be avoided?

**Comment by ladios** [comment]
Right, I forgot. #124 is needed. That one re-enables build deps installation.
